### PR TITLE
Configured NSIS and DEB generators for binary windows installer and d…

### PR DESCRIPTION
**Pull request by dkozinski**
…eb packages

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -75,3 +75,94 @@ if(NOT TARGET uninstall)
...
+
+# Package checksum
+set(CPACK_PACKAGE_CHECKSUM MD5)
+
+# Short description of the project
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY
+"eXtra-fast Essential Video Encoder, MPEG-5 EVC (Essential Video Coding), ${PACKAGE_PROFILE_NAME} profile")
+
+# Project homepage
+set(CPACK_PACKAGE_VENDOR "github.com/mpeg5/xeve")
```
Comment (dariusz-f) [inline CMakeLists.txt:114]: Set to "MPEG-5"

File: CMakeLists.txt [hunk 2]
```diff
@@ -75,3 +75,94 @@ if(NOT TARGET uninstall)
...
+set(CPACK_PACKAGE_HOMEPAGE_URL "https://github.com/mpeg5/xeve")
```
Comment (dariusz-f) [inline CMakeLists.txt:115]: Change url to github releases

File: CMakeLists.txt [hunk 3]
```diff
@@ -75,3 +75,94 @@ if(NOT TARGET uninstall)
...
+
+# Dependencies
+set(CPACK_COMPONENTS_ALL Runtime Libraries Development)
+set(CPACK_COMPONENT_DEVELOPMENT_DEPENDS Libraries)
+set(CPACK_COMPONENT_RUNTIME_DEPENDS Libraries)
+
+include(CPack)
+
+if( MSVC )
+  cpack_add_install_type(Full DISPLAY_NAME "Everything")
```
Comment (dariusz-f) [inline CMakeLists.txt:150]: Change to: "All"

File: CMakeLists.txt [hunk 4]
```diff
@@ -75,3 +75,94 @@ if(NOT TARGET uninstall)
...
+  cpack_add_install_type(Developer DISPLAY_NAME "Developer files")
+endif()
+
+cpack_add_component_group(DevelopmentGroup
+  DISPLAY_NAME "Development"
+  EXPANDED
+  DESCRIPTION
+  "All of the tools you'll ever need to develop software")
```
Comment (dariusz-f) [inline CMakeLists.txt:158]: Change this to be related with xeve, not general software development.

**Review by dariusz-f — APPROVED** [review]
looks good

**Review by kpchoi — APPROVED** [review]
LGTM, great

**Pull request merged**